docs/_static/mxnet-theme/index.html

```diff
@@ -3,11 +3,7 @@
     <div class="row">
       <div class="col-lg-12">
           <div id="banner-title" class='col-sm-6 col-xs-12'><span>Apache MXNet</span>
-            <p id="landing-title">is a flexible and efficient library for deep learning.</p>
-            <p id="intro">
```

Comment:
Why did you remove this? we still want it...

Reply:
This was per the previous PR I had done. - https://github.com/apache/incubator-mxnet-site/pull/33. 

Let me know when I have to add it back.

Reply:
👍🏼